### PR TITLE
mrconfig for mnc-python

### PR DESCRIPTION
**Pull request by caseyjlaw**
This is an example integration for mnc-python. I'd like this confirmed before I set up all the other deploy scripts and update here.

### CODE REVIEW

File: .mrconfig [hunk 1]
```diff
@@ -32,24 +32,13 @@ checkout =
...
     git submodule update
 
-[caltech-bifrost-dsp]
-checkout = 
-    git clone https://github.com/realtimeradio/caltech-bifrost-dsp.git
-    cd caltech-bifrost-dsp
-    git submodule init
-    git submodule update
-
-[bifrost]
```
Comment (rh-codebase) [inline .mrconfig:53]: what happened to bifrost?

File: .mrconfig_production [hunk 2]
```diff
@@ -34,7 +34,9 @@ checkout =
 
 [mnc_python]
 checkout =
-    git clone https://github.com/ovro-lwa/mnc_python.git
+    git pull --tags origin main
+    git checkout 0.8.0
+    git checkout -- .
```
Comment (rh-codebase) [inline .mrconfig_production:43]: what is this line doing? isn't 'git checkout 0.8.0 enough?

File: .mrconfig (hunk 1 above)
Comment (caseyjlaw) [reply .mrconfig:53]: I'd like to work our way up to the more complex builds. First step is a working deployment on cal-im with basic python.

File: .mrconfig_production (hunk 2 above)
Comment (caseyjlaw) [reply .mrconfig_production:43]: I copied syntax from your repos! I assumed there was something required to get tags set up right?

File: .mrconfig (hunk 1 above)
Comment (rh-codebase) [reply .mrconfig:53]: it's ok to pull the repo, if it doesn't have a deploy script then the service won't run it.

File: .mrconfig_production (hunk 2 above)
Comment (rh-codebase) [reply .mrconfig_production:43]: blindly copying code w/out understanding. :/ :) I should have added a comment here so I'll go hunt down why this line is necessary.

**Pull request closed without merging**